From d3f542cc9d0135a408be7ba995b8adad80bcd606 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Wed, 21 Aug 2019 14:06:29 +0200
Subject: [PATCH] travis: fetch only tags

The original `git pull` would try to pull & merge changes from the
master branch, which breaks CI runs on PRs based on non-master branches
---
 .travis.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.travis.yml b/.travis.yml
index f913db57..efb0df1b 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -29,7 +29,7 @@ before_script:
     sudo modprobe kvm-intel nested=1 || :
     sudo modprobe kvm-amd nested=1 || :
     dmesg | tail || :
-  - git pull --depth=100
+  - git fetch --tags --unshallow
   - |
     git describe --abbrev=0 --tags || :
     git describe --tags || :

